src/App/components/Alert/index.js

```diff
@@ -14,9 +14,9 @@ const Alert = ({ id, type, icon, close, text, className }) => {
             <i className="material-icons alert-icon" aria-hidden="true">{icon}</i>
             {text}{"\n"}
             {close ? <>
-                <a href="#" data-alert-close="" aria-label="Close alert" onClick={e => e.preventDefault()}>{"\n\t\t"}
+                <button data-alert-close="" onClick={e => e.preventDefault()}>{"\n\t\t"}
```

Comment:
This should maybe have aria-label to <button> after all?? 🤔

Reply:
The icon element does not have an aria-hidden attribute, and the icon name is "close". Is it then necessary?

Reply:
Probably not 👍

Reply:
RESOLVED